e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -296,13 +296,17 @@ def test_workspace_org() -> None:
         wait_for_experiment_state(test_exp_id, bindings.determinedexperimentv1State.STATE_COMPLETED)
         assert test_exp.projectId == default_project.id
 
-        # Moving an experiment out of the default project
+        # Move the test experiment into a user-made project
+        exp_count = len(bindings.get_GetProjectExperiments(sess, id=made_project.id).experiments)
+        assert exp_count == 0
         mbody = bindings.v1MoveExperimentRequest(
             destinationProjectId=made_project.id, experimentId=test_exp_id
         )
         bindings.post_MoveExperiment(sess, experimentId=test_exp_id, body=mbody)
         modified_exp = bindings.get_GetExperiment(sess, experimentId=test_exp_id).experiment
         assert modified_exp.projectId == made_project.id
+        exp_count = len(bindings.get_GetProjectExperiments(sess, id=made_project.id).experiments)
+        assert exp_count == 1
```

Comment:
It might also be good to assert that the moved project is no longer present in the uncategorized list - that's the issue I was running into.

Reply:
ok, updated the test